docker/Dockerfile

```diff
@@ -1,6 +1,24 @@
 # Docker image for running CF instances in the server.
 # Docker image includes HO(Host Orchestrator) inside,
 # so it could execute CF instance with API in HO.
+
+FROM debian:stable-20211011 AS cuttlefish-hostpkg-builder
```

Comment:
The commit message says:

```
Deprecate docker image for building host packages
This commit includes:
- Deprecate cuttlefish-hostpkg-builder docker image and migrate into
  cuttlefish-orchestration
```

And in this code we are still building the host packages. What do you mean by `Deprecate docker image for building host packages`?

Reply:
I'll modify the commit message. Sorry for confusing. I shouldn't use the term `Deprecate` for this commit.

Reply:
Remove the term `deprecate` from the commit message.